File: Wave Array[28-September-2023]/Solution.java
//  >> JAVA CODE <<


class Solution {
    public static void convertToWave(int n, int[] a) {
        for(int i=0;i<n-1;i+=2)
        {
            int temp= a[i];
            a[i]=a[i+1];
            a[i+1]=temp;
        }
    }
}

File: Wave Array[28-September-2023]/Solution.cpp
//  >> CPP CODE <<

class Solution{
    public:
    // arr: input array
    // n: size of array
    //Function to sort the array into a wave-like array.
    void convertToWave(int n, vector<int>& arr){
        
        for(int i=0;i<n-1;i+=2)
        {
            int temp=arr[i];
            arr[i]=arr[i+1];
            arr[i+1]=temp;
        }
        
        
    }
};